### PR TITLE
add changelog for v2.0.9

### PR DESCRIPTION
**Pull request by disksing**
### What problem does this PR solve? 
Prepare for releasing v2.0.9

### What is changed and how it works?
Add changelog for v2.0.9

### Check List 

Tests 
 - No code

### CODE REVIEW

**Comment by disksing** [comment]
/run-all-tests tidb=release-2.0 tikv=release-2.0 tidb-test=release-2.0

**Comment by disksing** [comment]
/run-unit-test tidb=release-2.0 tikv=release-2.0 tidb-test=release-2.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # PD Change Log
 
+## v2.0.9
+### Bug Fixes
+* Fix the issue that the PD server gets stuck caused by etcd startup failure [#1267](https://github.com/pingcap/pd/pull/1267)
+* Fix the issues related to `pd-ctl` reading the Region key [#1298](https://github.com/pingcap/pd/pull/1298) [#1299](https://github.com/pingcap/pd/pull/1299) [#1308](https://github.com/pingcap/pd/pull/1308)
```
Comment (rleungx) [inline CHANGELOG.md]: We also need to update the docs.

Comment (disksing) [reply CHANGELOG.md]: ok.

**Pull request merged**